en/getting_started/sensor_selection.md

```diff
@@ -1,61 +1,60 @@
 # Sensor Selection
 
-PX4 autopilot is capable of flying without any additional external
-sensors, because Pixhawk (the hardware FC) already has inbuilt sensors.
-There are 3 Gyros + 3 Accelerometers + 3 Magnetometers + Barometric
-pressure sensor for proper altitude reading. Despite that, we strongly
-recommend using **GPS** in all frames in order to unveil the full
-potential of the aircraft or vehicle and enable its automatic modes. For
-most of the fixed wing aircraft, **Airspeed sensor** is mandatory as
-well.
+PX4-based systems use sensors to determine vehicle state (state is required for stabilization and to enable autonomous control). The sensors provide information including: position/altitude, heading, speed, airspeed, orientation (attitude) and rates of rotation in different directions.
+
+The system *minimally requires* a gyroscope, accelerometer, magnetometer (compass) and barometer. A GPS or other positioning system is needed to enable all automatic [modes](../getting_started/flight_modes.md#categories), and some assisted modes. Fixed wing and VTOL-vehicles should additionally include an airspeed sensor (very highly recommended).
+
+The minimal set of sensors is incorporated into [Pixhawk Series](../flight_controller/pixhawk_series.md) flight controllers (and may also be in other controller platforms). Additional/external sensors can be attached to the controller.
+
+Below we describe some of the sensors. At the end there are links to information about [sensor wiring](#wiring).
+
 
 ## GPS
 
-There are many types of GPS you can use with Pixhawk and PX4. We
-recommend using GPS + Compass solution like the original 3DR GPS or
-alternative LEA 6H, NEO 7M, NEO 8 GPS+Compass solutions.
+Many types of GPS can be used with PX4. When using Pixhawk-series flight controllers we recommend using a *combined GPS + Compass*.
+
+> **Tip** The Pixhawk internal compass is close to other electronics, and may be susceptible to electromagnetic noise. An external compass/GPS mounted on a pedestal or wing (for fixed-wing) away from the rest of the system is recommended.
+
+Options include the original 3DR GPS, LEA 6H, NEO 7M, NEO 8 GPS+Compass solutions.

```

Comment:
- Are these options still valid/are there others?
- Can you provide a number of links to a number of recommended alternatives - 3DR GPS probably no longer even in existence. Some I found are below (but I don't know if these would work):
  * [ublox-neo-m8n-gps-with-compass](https://hobbyking.com/en_us/ublox-neo-m8n-gps-with-compass.html?gclid=Cj0KCQjwqM3VBRCwARIsAKcekb3ojv1ZhLz1-GuvCsUuGT8ZZuw8meMIV_I6pgUCj6DJRzHBY9OApekaAgI5EALw_wcB&gclsrc=aw.ds&___store=en_us)
  * [mRo GPS u-Blox Neo-M8N Dual Compass LIS3MDL+ IST8310](https://store.mrobotics.io/ProductDetails.asp?ProductCode=mro-gps003-mr)

Reply:
I think they're still valid. There are many options, for example (in addition to your's):
https://drotek.com/shop/en/184-u-blox
https://www.getfpv.com/holybro-micro-m8n-gps-module.html
https://www.getfpv.com/holybro-ublox-neo-m8n-gps-module.html
https://www.getfpv.com/here-gnss-gps-m8n.html
https://www.getfpv.com/3dr-ublox-gps-with-compass-kit.html
They're generally all compatible, but you have to be careful with the pinout. There are several different pin orderings.